index.js

```diff
@@ -155,7 +155,11 @@ SoundCloud.prototype.play = function (options) {
   } else if (this._playlist) {
     var length = this._playlist.tracks.length;
     if (length) {
-      this._playlistIndex = options.playlistIndex || 0;
+      if (options.playlistIndex === undefined) {
+        this._playlistIndex = this._playlistIndex || 0;
+      } else {
+        this._playlistIndex = options.playlistIndex;
+      }
```

Comment:
@ivangreene why not?
```
this._playlistIndex = options.playlistIndex || this._playlistIndex || 0;
```

Reply:
I originally had pretty much exactly this, but `.play({playlistIndex: 0})` would not be accepted as 0 is a falsey value.

Reply:
do you have another way that goes more with your code style so that passing `0` would still be acknowledged when `this._playlistIndex` is nonzero?